README.md

```diff
@@ -1,17 +1,58 @@
 Coclobas
-========
+======
 
-Configurable Cloudy Batch Scheduler
+Coclobas provides an OCaml interface on top of Kubernetes, making it easy to programatically interface with cluster from the command-line and from code. It provides logging facilities, maintaining logs long after Kubernetes disposes of them. Finally, it makes it easy to submit arbitrary script to be run in any Docker container, which makes it easier than using raw Kubernetes to submit arbitrarily complex jobs.
+
+Right now Coclobas is Kubernetes focused and works with Google Container Engine, but we aim to support arbitrary Kubernetes deployments, as well as alternate scheduler types.
 
 
 Build
 -----
 
-Uses the development version of Solvuu-build:
+Coclobas uses the development version of Solvuu-build:
 
     opam pin add solvuu-build https://github.com/solvuu/solvuu-build.git
 
-Then,
+Then, clode this repo and:
 
     make
+    
+After pinning `solvuu-build`, you can also just `opam pin add coclobas https://github.com/hammerlab/coclobas` to get going quickly.
+
+Requirements
+-----
+
+`kubectl` must be installed local to the Coclobas cluster, as it is shelled out to in order to query the Kubernetes cluster being used to run the submitted job.
+
+
+Using Coclobas
+-----
+
+You will need to briefly configure Coclobas prior to using it. You also will
+need to set up a Google Container Engine (GCE) cluster and note its name. This
```

Comment:
No, coclobas can/will create the cluster itself (eith with the `cluster-start` command or if needed when the server starts).

(When you aksed me for an existing cluster I thought it was because you didn't want to wait for the cluster creation)